assets/js/blocks/collection-filters/inner-blocks/price-filter/components/price-slider.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Internal dependencies
+ */
+import { EditProps } from '../types';
+import { getFormattedPrice } from '../utils';
+
+export const PriceSlider = ( { attributes, context }: EditProps ) => {
```

Comment:
Maybe its obvious from props but I think we should either call this EditPriceSlider or we should add a comment describing its purpose. Just a nitpick though.

Reply:
Let's add a comment explaining it. I'm just passing the props down to avoid create immediate types.

Reply:
A comment sounds good to me!

Reply:
Added in https://github.com/woocommerce/woocommerce-blocks/pull/11476/commits/e3a51f8e978e4ced61975bacd2540bd2105117c7